securedrop/config.py.example

```diff
@@ -78,9 +76,6 @@ GPG_KEY_DIR=os.path.join(SECUREDROP_DATA_ROOT, 'keys')
 TEMP_DIR = os.path.join(SECUREDROP_DATA_ROOT, "tmp")
 
 # Database configuration
-# TODO we currently use sqlite in production since it is sufficient and simple,
```

Comment:
this is a very old comment, back before we standardized on sqlite databases, so good to remove. #2225 is a good place for thoughts on this